.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "prettier.useTabs": true,
+  "prettier.tslintIntegration": true,
+  "prettier.trailingComma": "es5",
+  "prettier.stylelintIntegration": true,
+  "prettier.singleQuote": true,
+  "prettier.semi": false,
+  "prettier.eslintIntegration": true,
+  "prettier.printWidth": 800,
+  "editor.wordWrapColumn": 800
+}
```

Comment:
I like the idea of including the styling conventions used in this repo here. But I worry that it might be contentious. Happy to remove it.